ip/ffmpeg.c

```diff
@@ -37,7 +37,7 @@
 #endif
 
 #ifndef AVCODEC_MAX_AUDIO_FRAME_SIZE
-#define AVCODEC_MAX_AUDIO_FRAME_SIZE 192000
+#define AVCODEC_MAX_AUDIO_FRAME_SIZE 1920000
```

Comment:
To fit the additional frames I had to increase the size of this constant. I've increased it tenfold, but this obviously makes little sense. I don't have an idea how to approach this without changing too much code.